topo/node/srl/srl.go

```diff
@@ -94,6 +99,38 @@ func (n *Node) GenerateSelfSigned(ctx context.Context) error {
 	return n.cliConn.Close()
 }
 
+// ConfigPush pushes config lines proivded in r using scrapligo SendConfig
+func (n *Node) ConfigPush(ctx context.Context, r io.Reader) error {
+	log.Infof("%s - pushing config", n.Name())
+
+	cfg, err := io.ReadAll(r)
+	cfgs := string(cfg)
+
+	log.Debugf("config to push:\n%s", cfgs)
```

Comment:
not sure if we should log the entire cfgs content in case of a very large config. maybe its better to log the head of the config and the total size?

Reply:
I used the approach that currently exists in the repo.
In my opinion, observing the full config is beneficial as instructed by the user using the `-v debug` flag. The verbosity was requested.

Configs are indeed massive on SR Linux due to explicit ACLs, but when running without a debug they are not visible